manifests/m/Microsoft/VisualStudioCode/Insiders/1.99.0/Microsoft.VisualStudioCode.Insiders.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudioCode.Insiders
+PackageVersion: 1.99.0
+Commands:
+  - code-insiders
+InstallerType: inno
+Installers:
+  - Architecture: arm64
+    InstallerUrl: 'https://vscode.download.prss.microsoft.com/dbazure/download/insider/201df9b990f871fad842dc9a6211bfa84c663e53/VSCodeSetup-arm64-1.99.0-insider.exe'
+    InstallerSha256: 4067bacc20a8359bb5b03533fd99bcefd43e2a108ffb572930c545632f11058e
+    Scope: machine
+  - Architecture: arm64
+    InstallerUrl: 'https://vscode.download.prss.microsoft.com/dbazure/download/insider/201df9b990f871fad842dc9a6211bfa84c663e53/VSCodeUserSetup-arm64-1.99.0-insider.exe'
+    InstallerSha256: 80ecac2dceae1f882971758340896aa41a3e3bc1817e69b5294f8c13a58384f2
+    Scope: user
+  - Architecture: x64
+    InstallerUrl: 'https://vscode.download.prss.microsoft.com/dbazure/download/insider/6609ac3d66f4eade5cf376d1cb76f13985724bcb/VSCodeSetup-x64-1.98.0-insider.exe'
+    InstallerSha256: 72ff47fc4bc4bfc921a6774df13b563253f6ca793a888c5130e15bc25a66979b
+    Scope: machine
+  - Architecture: x64
+    InstallerUrl: 'https://vscode.download.prss.microsoft.com/dbazure/download/insider/6609ac3d66f4eade5cf376d1cb76f13985724bcb/VSCodeUserSetup-x64-1.98.0-insider.exe'
```

Comment:
These `InstallerUrl`s have the previous `PackageVersion` instead of the current.